cmd/panfs.go

```diff
@@ -1500,8 +1495,12 @@ func (fs *PANFSObjects) DeleteObject(ctx context.Context, bucket, object string,
 	// Delete the object.
 	if fs.isConfigAgentObject(bucket, object) {
 		objectName := pathJoin(bucket, object)
-		noLock := ""
-		err = fs.configAgent.DeleteObject(objectName, noLock)
+		if opts.DeletePrefix {
+			err = fs.configAgent.DeleteObjectsByPrefix(objectName)
```

Comment:
even despite the fact that we should not go to this branch anywhere in our code (we seems checked IAM yesterday), I'd vote for keeping this change just in case, at it should not make things worse.

Reply:
Although, bucket metadata code seems MAY try to read old versions of metadata files, so we may have some read requests to this location, which we can handle in upper level and always return "not found" error.